extract/html-table/README.md

```
@@ -0,0 +1,61 @@
+- `handleRowspan` - `boolean` - (optional): Determines how to handle rowspan. Default is true.
+- `maxDepth` - `number` - (optional): Maximum depth for nested tables. Default is 3.
+- `debug` - `boolean` - (optional): Enables debug logging. Default is false.
+
+## API Calls
+
+- `api.files.download`
+- `api.files.update`
+
+## Usage
+
+**install**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)